### DISCUSSION TITLE
A way to override which modules are enabled as a launch arguments.

### DISCUSSION THREAD

**Discussion by DanielWillett** [SmartlyDressedGames/Unturned-3.x-Community#4295] [Ideas]
This will allow us to make desktop shortcuts to launch with specific modules instead of having to modify the configs for each one every time. It also allows you to run multiple servers with different modules off the same installation.

Example:
`.\Unturned.exe -EnableModules "Module1;Module2;Module3" -DisableModules "Module4;Module5"`

or maybe just only enable the listed modules if the launch argument is present and don't load any others.
`.\Unturned.exe -Modules "Module1;Module2;Module3"`

**Comment by surv0013** [comment]
Try this:
`.\Unturned.exe -disableModule/Module0 -disableModule/Module1 -disableModule/Module2`

**Reply by SDGNelson** [reply]
Thanks surv0013. Yes, this is the supported approach.

**Reply by DanielWillett** [reply]
mb, idk how i missed that. thanks
